lib/chef/handler/datadog.rb

```diff
@@ -120,6 +121,67 @@ def restore_env_proxies
         ENV['http_proxy'] = @env_http_proxy
         ENV['https_proxy'] = @env_https_proxy
       end
+
+      # create and configure all the Dogapi Clients to be used
+      #
+      # @return [Array] all Dogapi::Client to be used
+      def prepare_the_pack
+        dogs = []
+        endpoints.each do |url, api_key, app_key|
+          dogs.push(Dogapi::Client.new(
+                      api_key,
+                      app_key,
+                      nil,   # host
+                      nil,   # device
+                      true,  # silent
+                      nil,   # timeout
+                      url
+          ))
+        end
+        dogs
+      end
+
+      def convert_to_array(value_or_array)
+        return value_or_array if value_or_array.nil?
+        value_or_array.is_a?(Array) ? value_or_array : [value_or_array]
+      end
+
+      # return all endpoints as a list of triplets [url, api_key, application_key]
+      def endpoints
+        validate_keys(@config[:api_key], @config[:application_key], true)
+        @config[:url] ||= ENV['DATADOG_HOST']
```

Comment:
why have you added this fallback to `ENV['DATADOG_HOST']`? Shouldn't we just make sure it's set to nil if it's a "falsey" value, and let `dogapi` decide?